pydantic/analyzed_type.py

```diff
@@ -130,26 +132,28 @@ def __init__(self, __type: Any, *, config: ConfigDict | None = None, _parent_dep
             core_schema = _get_schema(__type, core_config, parent_depth=_parent_depth + 1)
 
         validator: SchemaValidator
-        try:
+        if hasattr(__type, '__pydantic_validator__') and config is None:
             validator = __type.__pydantic_validator__
-        except AttributeError:
+        else:
             validator = SchemaValidator(core_schema, core_config)
 
         serializer: SchemaSerializer
-        try:
+        if hasattr(__type, '__pydantic_serializer__') and config is None:
             serializer = __type.__pydantic_serializer__
-        except AttributeError:
+        else:
             serializer = SchemaSerializer(core_schema, core_config)
 
         self.core_schema = core_schema
         self.validator = validator
         self.serializer = serializer
 
-    def validate_python(self, __object: Any) -> T:
-        return self.validator.validate_python(__object)
+    def validate_python(self, __object: Any, *, strict: bool | None = None, context: dict[str, Any] | None = None) -> T:
+        return self.validator.validate_python(__object, strict=strict, context=context)
```

Comment:
need to check whether `context=None` get's passed through as a value, e.g. is different to omitting context.

Reply:
Should they be different? I thought it was None by default

Reply:
I confirmed that it's the same as far as the Python side is concerned